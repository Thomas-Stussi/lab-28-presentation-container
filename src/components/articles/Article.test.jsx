import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Article from './Article';

describe('Article component', () => {
  afterEach(() => cleanup());
  it('render Article component', () => {
    const { asFragment } = render(<Article
      title="title"
      author="author"
      description="description"
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});
