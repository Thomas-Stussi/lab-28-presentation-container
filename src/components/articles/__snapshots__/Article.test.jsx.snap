// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article component render Article component 1`] = `
<DocumentFragment>
  <h3>
    title
  </h3>
  <p>
    By: author
  </p>
  <p>
    Summary: description
  </p>
</DocumentFragment>
`;
